diff --git a/node_modules/@preconstruct/hook/index.js b/node_modules/@preconstruct/hook/index.js
index 66e6cc4..fe062e5 100644
--- a/node_modules/@preconstruct/hook/index.js
+++ b/node_modules/@preconstruct/hook/index.js
@@ -7,6 +7,7 @@ let EXTENSIONS = [".js", ".jsx", ".ts", ".tsx"];
 
 let babelPlugins = [
   require.resolve("@babel/plugin-transform-modules-commonjs"),
+  require.resolve("babel-plugin-transform-import-meta"),
 ];
 
 exports.___internalHook = (distDir, relativeToRoot, relativeToPkgDir) => {
